### PR TITLE
Add custom copy for google oauth provider

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by gewenyu99** [comment]
Screenshot 2023-05-24 at 12 10 03 PM
I think the only thing that's different is `... and select web application as the application type.`

**Review by ArmanNik — COMMENTED** [review]
@gewenyu99 I might have missed it but how is this component different from the `mainOAuth` component that is the default for all oauths?

File: src/routes/console/project-[project]/auth/googleOAuth.svelte [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <InputPassword
+            id="secret"
+            label="App Secret"
+            placeholder="Enter App Secret"
+            minlength={0}
+            showPasswordButton
+            bind:value={secret} />
+        <Alert type="info">
+            To complete set up, add this OAuth2 redirect URI to your {provider.name} app configuration
+            and select web application as the application type.
```
Comment (stnguyen90) [inline src/routes/console/project-[project]/auth/googleOAuth.svelte:62]: You would actually create a OAuth2 client ID w/ Web application as the Application type first:

image

Then, set the redirect URIs:

image

Comment (gewenyu99) [reply src/routes/console/project-[project]/auth/googleOAuth.svelte:62]: I see, let me change

**Comment by gewenyu99** [comment]
> @gewenyu99 I might have missed it but how is this component different from the `mainOAuth` component that is the default for all oauths?

@ArmanNik We need to mention that they need to create OAuth as a web app even for mobile and flutter.

**Review by stnguyen90 — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**